script.js

```diff
@@ -47,6 +64,140 @@ hamburger.onclick = () => {
   }
 };
 
+const projects = [
+  {
+    name: 'Multi-Post Stories Gain+Glory',
+    description:
+      'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Et laborum sunt ipsa molestias odit, accusamus iste labore doloribus cum enim est ea perferendis, laudantium eum obcaecati natus consequatur nisi voluptatibus. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Et laborum sunt ipsa molestias odit, accusamus iste labore doloribus cum enim est ea perferendis, laudantium eum obcaecati natus consequatur nisi voluptatibus. Lorem ipsum dolor sit amet, consectetur adipisicing elit. Et laborum sunt ipsa molestias odit, accusamus iste labore doloribus cum enim est ea perferendis, laudantium eum obcaecati natus consequatur nisi voluptatibus.',
+    tags: ['Ruby on rails', 'css', 'JavaScript', 'html'],
+    images: [
+      './assets/background/desktop-project.png',
+      './assets/background/mobile-project.png',
+    ],
+    sourceLink: '#',
+    demoLink: '#',
```

Comment:
[OPTIONAL] Why not make the project more realistic 🤔 by adding real links for these? Perhaps a link to your live portfolio for the live and a link to the GitHub repo for the source 🔗 . It can be the same link for all of them as these are dummy projects. 😉